modules/sdk/project-templates/project-templates/src/main/java/com/liferay/project/templates/internal/Archetyper.java

```diff
@@ -72,6 +79,33 @@
  */
 public class Archetyper {
 
+	public static String getManifestProperty(File file, String property)
+		throws IOException {
+
+		try (JarFile jarFile = new JarFile(file)) {
+			Manifest manifest = jarFile.getManifest();
+
+			Attributes attributes = manifest.getMainAttributes();
+
+			return attributes.getValue(property);
+		}
+	}
+
+	public static String getTemplateName(String name) {
```

Comment:
Same thing, can we move it to `FileUtil` or a `ProjectTemplatesUtil`? Just to keep `Archetyper` clean, what do you think? 😊